src/main/java/com/criteo/vips/enums/VipsImageFormat.java

```diff
@@ -14,17 +14,17 @@
   limitations under the License.
 */
 
-package com.criteo.vips;
+package com.criteo.vips.enums;
 
-public enum ImageFormat {
+public enum VipsImageFormat {
```

Comment:
Isn't this enumeration generated?

Reply:
No, it doesn't. This is just a custom enum to wrap image string extensions.